src/api.js

```diff
@@ -11,3 +11,12 @@ export const sendHeartbeat = async (user) => {
   }
   return await fetch('/heartbeat', requestOptions)
 }
+
+export const getStartedGame = async (playerId) => {
+  const response = await fetch(`/my-started-game?playerId=${playerId}`)
+
+  if (response.status === 404) console.log('NOT FOUND started game')
```

Comment:
This is a good start, but when we get a 404 response our body is empty, so the lines below will cause an Error to be thrown.